.env.example

```diff
@@ -1,5 +1,7 @@
 NODE_ENV=development
 WP_URL=http://direct.faforever.com/wp-json
+WP_NEWSHUB_CATEGORYID=0
+WP_NEWSHUBARCHIVE_CATEGORYID=0
```

Comment:
env variables need to be added in stack .env files as well

Reply:
this is in progress here
https://github.com/FAForever/faf-stack/pull/146